app/services/teachers/induction_period.rb

```diff
@@ -9,6 +9,22 @@ def induction_start_date
     first_induction_period&.started_on
   end
 
+  def formatted_induction_start_date
+    induction_start_date&.to_fs(:govuk)
+  end
+
+  def induction_programme
+    return unless first_induction_period
+
+    ::INDUCTION_PROGRAMMES[first_induction_period.induction_programme.to_sym]
+  end
+
+  def appropriate_body_name
+    return unless first_induction_period
+
+    first_induction_period.appropriate_body.name
```

Comment:
Are we getting the first_induction_period based on `started_on` order?
Shouldn't it be the last instead?